### PR TITLE
dMRI tweaks

### PR DESCRIPTION
**Pull request by mharms**
Fixes a couple minor bugs involving the creation of the `bvecs_noRot` file, and adds comments to explain code.

### CODE REVIEW

**Comment by mharms** [comment]
@MichielCottaar: Please review carefully. Proposed changes have not been tested.

**Review by glasserm — APPROVED** [review]

**Comment by MichielCottaar** [comment]
Thanks for this. It looks correct to me. I'd be happy to run some tests using all configurations of SelectBestB0 and CombineDataFlag to be sure. However, I'm moving flats today, so I won't have time to do that until later in the week.

**Comment by mharms** [comment]
@MichielCottaar That would be great.  Thx.

**Comment by glasserm** [comment]
Should this be merged?

**Comment by mharms** [comment]
I was waiting to hear from @MichielCottaar regarding his tests of all combinations of SelectBestB0 and CombineDataFlag.

File: DiffusionPreprocessing/scripts/eddy_postproc.sh [hunk 1]
```diff
@@ -41,40 +41,47 @@ qc_command+=(-v)
...
 if [ ${CombineDataFlag} -eq 2 ]; then
-	# remove first volume as this is the reference b0, which was added to the dataset before running eddy
+	
 	if [ ${SelectBestB0} -eq 1 ]; then
+		# remove first volume/value as this reflects the "best b0", which was added to the dataset before running eddy
 		${FSLDIR}/bin/fslroi ${eddydir}/eddy_unwarped_images ${datadir}/data 1 -1
-		cut -d' ' -f2- ${eddydir}/eddy_unwarped_images.eddy_rotated_bvecs ${datadir}/bvecs # removes first value from bvecs
+		cut -d' ' -f2- ${eddydir}/Pos_Neg.bvals >${datadir}/bvals
+		cut -d' ' -f2- ${eddydir}/Pos_Neg.bvecs >${datadir}/bvecs_noRot
+		cut -d' ' -f2- ${eddydir}/eddy_unwarped_images.eddy_rotated_bvecs ${datadir}/bvecs
```
Comment (MichielCottaar) [inline DiffusionPreprocessing/scripts/eddy_postproc.sh:54]:
```suggestion
		cut -d' ' -f2- ${eddydir}/eddy_unwarped_images.eddy_rotated_bvecs >${datadir}/bvecs
```

**Comment by MichielCottaar** [comment]
With the suggested fix the pipeline works for data acquired fully with both phase encode directions (both with and without --select-best-b0 and for --combine-data-flag set to 0, 1, or 2). I just started to test on a dataset with much of the data acquired with only a single phase encode direction. I should be able to get that done by tomorrow. Any fixes could be in this pull request or a new one if you prefer to merge this one.

**Comment by mharms** [comment]
Happy to make any necessary changes here, if you find anything additional in the ongoing tests.

**Comment by MichielCottaar** [comment]
#202 is a pull request unto the dMRI_tweaks rather than the master branch. Actually supporting the CombineDataFlag of 0 required some larger changes, so this seemed the easiest way to get it into this pull request.

**Comment by glasserm** [comment]
Should this also be merged?

**Pull request merged**